src/Entities/Tetrahedrons.jl

```diff
@@ -48,22 +49,28 @@ function create_entity(t::Tetrahedron, vn::AbstractVector{<:AbstractNode})
     Tetrahedron(vn, label(t))
 end
 
+"Contains the cache to compute the element internal forces and stiffness matrix."
 struct TetrahedronCache{T} <: AbstractElementCache
     fint::Vector{T}
-    # TODO Use Symmetric K.
     Ks::Matrix{T}
     σ::Matrix{T}
     ε::Matrix{T}
     F::Matrix{T}
     H::Matrix{T}
+    X::Matrix{T}
+    J::Matrix{T}
+    funder::Matrix{T}
     function TetrahedronCache()
         fint = zeros(12)
-        Ks = zeros(12, 12)
-        σ = zeros(3, 3)
-        ε = zeros(3, 3)
+        Ks = Symmetric(zeros(12, 12))
```

Comment:
This is not effective since `Ks` is annotated as a `Matrix` above.

Reply:
Done

Reply:
Done